design_handoff_mission_control/prototype/v1-conservative.jsx

```
@@ -0,0 +1,621 @@
+        borderBottom:`1px solid ${V1.border}`, background:V1.surface,
+      }}>
+        <div style={{ display:'flex', alignItems:'center', gap:8 }}>
+          <OrbitStar size={18} color={V1.accent} />
+          <span style={{ fontWeight:600, fontSize:14, letterSpacing:'-0.01em' }}>Mission Control</span>
+        </div>
+        <span style={{ color:V1.fgMuted, fontFamily:V1.mono, fontSize:11 }}>WED · 2026-05-27 · 09:12</span>
+
+        <nav style={{ marginLeft:18, display:'flex', gap:4, alignItems:'center' }}>
+          {['overview','insights','review'].map(t => (
+            <button key={t} onClick={() => setTab(t)} style={{
+              padding:'5px 10px', fontSize:12, fontWeight:500, textTransform:'capitalize',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)